pyccl/halos/halo_model.py

```diff
@@ -1744,7 +1743,7 @@ def get_kr_f2_f2T_X():
     if np.ndim(k) == 0:
         out = np.squeeze(out, axis=-1)
         out = np.squeeze(out, axis=-1)
-    return out
+    return t1113 + t1122 # out
```

Comment:
shouldn't this be outputting `out`?  You need to multiply by the `i1`, `i2`, etc. (see Eq 3.25 of https://arxiv.org/pdf/1912.08209.pdf) to the trispectrum. Do you know why you can reproduce the external code's results without these? Furthermore, you need to make sure the output has the right dimensions given the input `a` and `k`.